Client/Frontend/Home/ASLibraryCell.swift

```diff
@@ -16,10 +16,18 @@ class ASLibraryCell: UICollectionViewCell, NotificationThemeable {
 
     var libraryButtons: [LibraryShortcutView] = []
 
-    let bookmarks = LibraryPanel(title: Strings.AppMenuBookmarksTitleString, image: UIImage.templateImageNamed("menu-Bookmark"), color: UIColor.Photon.Blue40)
-    let history = LibraryPanel(title: Strings.AppMenuHistoryTitleString, image: UIImage.templateImageNamed("menu-panel-History"), color: UIColor.Photon.Violet50)
-    let readingList = LibraryPanel(title: Strings.AppMenuReadingListTitleString, image: UIImage.templateImageNamed("menu-panel-ReadingList"), color: UIColor.Photon.Pink40)
-    let downloads = LibraryPanel(title: Strings.AppMenuDownloadsTitleString, image: UIImage.templateImageNamed("menu-panel-Downloads"), color: UIColor.Photon.Green60)
+    let bookmarks = LibraryPanel(title: .AppMenuBookmarksTitleString,
+                                 image: UIImage.templateImageNamed("menu-Bookmark"),
+                                 color: UIColor.Photon.Blue40)
+    let history = LibraryPanel(title: .AppMenuHistoryTitleString,
+                               image: UIImage.templateImageNamed("menu-panel-History"),
+                               color: UIColor.Photon.Violet50)
+    let readingList = LibraryPanel(title: .AppMenuReadingListTitleString,
+                                   image: UIImage.templateImageNamed("menu-panel-ReadingList"),
+                                   color: UIColor.Photon.Pink40)
+    let downloads = LibraryPanel(title: .AppMenuDownloadsTitleString,
+                                 image: UIImage.templateImageNamed("menu-panel-Downloads"),
+                                 color: UIColor.Photon.Green60)
```

Comment:
copy & pasting my comment from above: 

I'm gonna make the argument that this is much more easily readable than a super long line of code. We should update our style to this format... or really, we should technically be doing it this way already, if we're currently following the Wenderlich style guide, because this is part of that style.

Sure, we lose searchability, but you never search for title: .AppMenuCloseAllTablTitleString, style: .default, handler:. I'm merely updating the style to reflect what should be in usage on parts that I'm actually touching. :)

Reply:
I prefer having it on multiple lines as well. Easier to read